### ISSUE TITLE
How to find this both details of modules  SECRET $BUILDSECRET 

### ISSUE THREAD

**Issue by kuttysoftmy** [jesussuarz/bypass-license-modulesgarden#2]
How to find this both details of modules if we know this 2 details we can make keygen also for them like host file editing method 

but how to find them that is the challange 

$SECRET = "*******************************";          # 
$BUILDSECRET = "**************";

**Comment by wbhoster** [comment]
yeah that is right because i have also almost all modules in untouched version but don't have these two things
